### ISSUE TITLE
InvertedDoublePendulumEnv is broken

### ISSUE THREAD

**Issue by ryanjulian** [rlworkgroup/garage#68]
```sh
Traceback (most recent call last):
  File "tests/envs/test_envs.py", line 65, in <module>
    envs = [cls() for cls in simple_env_classes]
  File "tests/envs/test_envs.py", line 65, in <listcomp>
    envs = [cls() for cls in simple_env_classes]
  File "/home/rjulian/code/garage/rllab/envs/mujoco/point_env.py", line 21, in __init__
    super(PointEnv, self).__init__(*args, **kwargs)
  File "/home/rjulian/code/garage/rllab/envs/mujoco/mujoco_env.py", line 85, in __init__
    self.reset()
  File "/home/rjulian/code/garage/rllab/envs/mujoco/mujoco_env.py", line 131, in reset
    return self.get_current_obs()
  File "/home/rjulian/code/garage/rllab/envs/mujoco/mujoco_env.py", line 134, in get_current_obs
    return self._get_full_obs()
  File "/home/rjulian/code/garage/rllab/envs/mujoco/mujoco_env.py", line 138, in _get_full_obs
    cdists = np.copy(self.sim.geom_margin).flat
AttributeError: 'mujoco_py.cymj.MjSim' object has no attribute 'geom_margin'
```

**Comment by ryanjulian** [comment]
```sh
Traceback (most recent call last):
  File "tests/envs/test_envs.py", line 65, in <module>
    envs = [cls() for cls in simple_env_classes]
  File "tests/envs/test_envs.py", line 65, in <listcomp>
    envs = [cls() for cls in simple_env_classes]
  File "/home/rjulian/code/garage/rllab/envs/mujoco/inverted_double_pendulum_env.py", line 21, in __init__
    super(InvertedDoublePendulumEnv, self).__init__(*args, **kwargs)
  File "/home/rjulian/code/garage/rllab/envs/mujoco/mujoco_env.py", line 85, in __init__
    self.reset()
  File "/home/rjulian/code/garage/rllab/envs/mujoco/mujoco_env.py", line 127, in reset
    self.reset_mujoco(init_state)
  File "/home/rjulian/code/garage/rllab/envs/mujoco/inverted_double_pendulum_env.py", line 53, in reset_mujoco
    self.sim.data.qpos = qpos
AttributeError: attribute 'qpos' of 'mujoco_py.cymj.PyMjData' objects is not writable
```

**Comment by ryanjulian** [comment]
Fixed in #58
